turborepo-tests/examples/README.md

```diff
@@ -0,0 +1,27 @@
+# Turborepo Examples tests
+
+These tests are for ensuring that the examples we are providing to users work right out of the box.
+
+## Test structure
+
+We guarantee that the examples work by running the `lint` and `build` tasks in each example and checking for a `>>> FULL TURBO` on the second run. This requires that the tasks pass and are cacheable. The logic for the tests is in `test_example.sh` and takes a few steps:
+
+1. `cd` to the example directory.
+2. Install packages.
+3. Run `turbo build lint`.
+4. Run `turbo build lint` again and write the log results to a temporary text file.
+5. Read that text file looking for a `>>> FULL TURBO`.
```

Comment:
This is basically the same as it was before, minus prysk right?

Reply:
Correct! The major spiritual change is that we're not doing true snapshot tests. Rather, we're just checking for some "FULL TURBO".